caikit/core/model_management/local_model_trainer.py

```diff
@@ -173,7 +176,6 @@ def wait(self):
 
             # If running a subprocess, handle abnormal exit codes
             if self._use_subprocess:
-                self._completion_time = self._worker.get_or_throw()
```

Comment:
I think calling `get_or_throw` here was a bug.. @gabe-l-hart sound correct?

Reply:
Nope! This is how the completion time gets communicated from the subprocess back to the parent. This is needed for the retention policy stuff, otherwise subprocess-based trainings will live forever

Reply:
It's quite possible that this does not play nicely with errors, though

Reply:
hmm. the destroyable process seems to be returning error string or throw error for the object `_result` returned by `get_or_throw()` 🤔